src/index.html

```diff
@@ -4,16 +4,19 @@
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<% if (process.env.NODE_ENV==="production" ) { %>
-		<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/;  img-src 'self' ">
+	<% if (process.env.NODE_ENV === "production" ) { %>
+		<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' 'unsafe-eval' blob; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/;  img-src 'self';worker-src 'self' blob: ">
 	<% } %>
-	<title>JSRL</title>
+	<title><%= htmlWebpackPlugin.options.title %></title>
 	<link href="./unicodetiles/unicodetiles.css" rel="stylesheet" type="text/css" />
 	<script src="./unicodetiles/unicodetiles.js"></script>
```

Comment:
I'm going to refactor stuff so we are not tied to this dependency for pixi deployments, but we can fix that later.

Reply:
cool